LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+Copyright (c) 2015-2016 Benjy Cui
```

Comment:
Should be Alipay, see: https://github.com/ant-design/ant-design/blob/develop-1.0.0/LICENSE